lib/routes/account.js

```diff
@@ -497,6 +497,14 @@ module.exports = function (
           return db.emailRecord(email)
             .then(
               function (result) {
+
+                // Check to see if the correct email case was used and don't explicitly
+                // flag as an incorrect password attempt because the case was not
+                // the same as what they used to sign-up with.
```

Comment:
The code change looks fine, but I feel like this comment probably won't help us very much in a few months time, as it doesn't remind what's the bad thing that happens if we flag it.

I suggest something like:

"""
If the incorrect email case is used, the password check cannot possibly succeed,
and the client will retry automatically with the correct capitalization.
Don't tell customs-server this was a failed login attempt, to avoid penalizing the user twice.
"""